cypress-base.config.ts

```diff
@@ -2,7 +2,7 @@ import { defineConfig } from "cypress";
 
 const baseConfig = defineConfig({
   e2e: {
-    baseUrl: 'http://localhost:3000',
+    baseUrl: 'http://localhost:3001',
```

Comment:
You should NOT change this config, what is the purpose for this?

Reply:
i did it so the cypress test would run, if i havent change that the test couldent run

Reply:
That means you are not running the tests correctly. In the [README file](https://github.com/devxicans/platform?tab=readme-ov-file#monorepo-commands) the commands there is 1 specific for running the tests:

```
pnpm test:cypress:dev --filter=@1xdev/APPLICATION - Run cypress tests in a specific application.
This will start up the application and run cypress on it.
```

If you need to modify this config file to point to port `3001` that means you were running both applications on dev and not using the cypress command.